app/controllers/account/disability_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Account
```

Comment:
Yeri gelmişken ve kimlik yapısını değiştirirken karşılaştığım bir problemi burada dile getireyim.

Bu ve buna benzer controller'ları `Account` modülünün altına almasak daha iyi olacak, `Account` modülü bana, login olan kullanıcının yani `current_user` ile ilgili ayarların yapılacağı kontroller ve işlevleri barındıran bir namespace olarak akla geliyor.

Engellik durumu güncellemesini ele alırsak, bu işlemi sadece yetkili bir kullanıcı yapabilir yani kullanıcıların yönetileceği ve kullanıcılarla ilgili her türlü değişikliğin yapılabileceği bir modül altına almak daha mantıklı görünüyor. Benim ilk başta aklıma gelen modül ismi `UserManagement` tabi düşünüp güzel bir namespace belirlersek daha iyi olur.

Reply:
Haklısın, bunu daha önce de konuşmuştuk. Bununla ilgili #749 iş kaydı da var. O yüzden bahsettiğin düzenlemeyi bu PR'da yapmayalım derim.

Reply:
Sadece iş kaydı değil, @msdundar'ın başladığı bir branch bile var: https://github.com/omu/nokul/commits/refactor-accounts

Reply:
Ayrıca https://github.com/omu/nokul/issues/750

@ecmelkytz'a katılıyorum.  Bu başka bir PR'ın konusu ve gördüğüm kadarıyla ciddi bir refactoring yapılmalı.